capstone-frontend/src/components/ClubGridItem.jsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
+import { Card } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
 
 import '../styles/Groups.css';
 
-const ClubGridItem = ({id, name, description, ownerID, gbookID}) => {
+const ClubGridItem = ({club}) => {
+  const header = club.ownerID === window.localStorage.getItem("userID") ?
+                     <Card.Header className="header text-muted"> Owner </Card.Header> :
+                     <Card.Header className="header text-muted"> Member </Card.Header>;
```

Comment:
I like the card headers on these tiles!

Reply:
If we are going to have a Club Search page, is there a way to make this more dynamic to leave out `Owner` or `Member` in header in the case that someone is simply viewing the club?

Reply:
Thanks! And that's a good idea. Maybe I can have a third header for "Viewers".

Reply:
I didn't think to include it here since this page will only show `Clubs` that you are either a member of or an owner of. But if we use these same components elsewhere this would be a good thing to include.